test/solidity/Periphery/Permit2Proxy.t.sol

```diff
@@ -542,6 +569,30 @@ contract Permit2ProxyTest is TestBase {
         return testdata;
     }
 
+    function test_RevertOnCustomErrorAndAllowance() public {
```

Comment:
test naming is inconsistent with our guidelines.
Should be testRevert_....

Reply:
updated